addons/account/models/account_move_line.py

```diff
@@ -1136,7 +1136,7 @@ def _inverse_analytic_distribution(self):
     @api.onchange('account_id')
     def _inverse_account_id(self):
         self._conditional_add_to_compute('tax_ids', lambda line: (
-            line.account_id.tax_ids
+            line.account_id
```

Comment:
I don't understand this change

Reply:
Because with this, when we get the suggestion but we don't want it and we select another account_id we won't get the taxes from the suggestion. You gave me the ok, in the discord thread If you want to check out the discussion. 🙂

Reply:
That's one month old, I've seen about a hundred of PRs since; my memory has some limits :)

Reply:
Yes I know, that's why I told you about the thread 🙂, but sorry for the poor explanation. I checked everything again, and It actually seems to work as we expect with the `.tax_ids` so I just left the change we agreed in the first place. To only show the suggestion if the 2 last lines are the same.